schemas/label.json

```diff
@@ -0,0 +1,104 @@
+{
+  "$id": "https://json-schema.org/draft/2019-09/schema",
+  "type": "object",
+  "default": {},
+  "title": "Llamafolio Label Schema",
```

Comment:
Hey @eabz I know it's just a validation file, but let's keep the naming of the `LlamaFolio` in `PascalCase` :)

Can you update on `Readme.md` too please?